docs/disclosure-log.md

```diff
@@ -94,3 +96,253 @@ This action does not enable because the light client on `B` has a more recent co
 Chain `B` drops this message.
 
 From this point on, the model stutters, i.e., is unable to progress further in the connection handshake protocol.
+
+
+### 3. ICS3 problems due to version negotiation
+
+__Context__.
+The original issue triggering this discussion is here: [cosmos/ics/#459](https://github.com/cosmos/ics/issues/459).
+Briefly, version negotiation in the ICS3 handshake can interfere in various ways, breaking either the safety or liveness of this protocol.
```

Comment:
The current liveness issues are expected as the two parties do not agree on a version. Not sure if and how these should be reported. If there would be liveness issue due to protocol bugs (e.g. the two parties do not agree even if there is one common version) the format here would be fine.